<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPEStub</class>
 <widget class="QDialog" name="DialogPEStub">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>150</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Stub</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="labelOffset">
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditOffset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButtonX" name="pushButtonRich">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Rich</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBoxX" name="checkBoxReadOnly">
       <property name="text">
        <string>Read only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonHex">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButtonX" name="pushButtonAdd">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonReplace">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonRemove">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonDump">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Dump</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEditX" name="lineEditStatus">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
